app/app.rb

```diff
@@ -0,0 +1,84 @@
+ENV["RACK_ENV"] ||= "development"
+
+require 'sinatra/base'
+require 'sinatra/flash'
+require_relative 'models/user'
+require_relative 'models/post'
+require_relative 'data_mapper_setup'
+
+
+class Chitter < Sinatra::Base
```

Comment:
Good refactor step would be to split each function into separate controller files:
https://github.com/makersacademy/chitter-challenge/blob/master/docs/review.md#split-routes-into-separate-controller-files